VirtualListView/PositionalViewSelector.cs

```diff
@@ -30,6 +30,10 @@ int GetTotalCount()
 
 		var hasAtLeastOneItem = false;
 		var numberOfSections = Adapter.GetNumberOfSections();
+		if (Adapter is null)
```

Comment:
This seems innocent enough, however it should be moved above the line of code it's under... really to the start of the method.  You're checking for Adapter being null after the call to `Adapter.GetNumberOfSections()`...

Reply:
yeah sounds right :)

Reply:
fixed it